core/domain/exp_domain.py

```diff
@@ -777,8 +994,29 @@ def from_dict(cls, interaction_dict):
 
     def __init__(
             self, interaction_id, customization_args, answer_groups,
+<<<<<<< HEAD
```

Comment:
Yes, sorry I resolved alredy this merge problem after the PR ;)
And I have added docstring for new classes and methods.